pom.xml

```diff
@@ -1657,8 +1658,8 @@
             <id>Coverage</id>
             <dependencies>
                 <dependency>
-                    <groupId>net.sourceforge.cobertura</groupId>
-                    <artifactId>cobertura</artifactId>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>cobertura-maven-plugin</artifactId>
                     <version>${cobertura.version}</version>
```

Comment:
Why not the latest version `2.6` of the plugin: http://search.maven.org/#artifactdetails%7Corg.codehaus.mojo%7Ccobertura-maven-plugin%7C2.6%7Cmaven-plugin